### PR TITLE
Add images for Tenant architecture, Console dashboard

### PR DESCRIPTION
**Pull request by ravindk89**
Small update to add images for architecture + console.

Can move the images to `.github` as well. I've reduced res, but can probably push it a bit farther before quality drops.

### CODE REVIEW

**Review by eco-minio — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Each MinIO Tenant represents an independent MinIO Object Store within
 the Kubernetes cluster. The following diagram describes the architecture of a
 MinIO Tenant deployed into Kubernetes:
 
-IMAGE
+<img src=docs/images/architecture.png width="600px"/>
```
Comment (harshavardhana) [inline README.md:29]:
```suggestion
<img src=docs/images/architecture.png width="600px"/>
```

You should use markdown syntax here @ravindk89 using HTML is not advised generally

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ MinIO provides multiple methods for accessing and managing the MinIO Tenant:
 The MinIO Console provides a graphical user interface (GUI) for interacting with
 MinIO Tenants. 
 
-IMAGE
+<img src=docs/images/console-dashboard.png width="600px"/>
```
Comment (harshavardhana) [inline README.md:38]: You should use markdown syntax here @ravindk89 using HTML is not advised generally

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by ravindk89** [comment]
@harshavardhana I think the markdown syntax doesn't let you set width for the images, so they'll be a bit wide. I can just resize them to 600px though, so it should be fine. Will update the PR shortly.

**Comment by ravindk89** [comment]
@harshavardhana addressed your concerns.

File: README.md (hunk 1 above)
Comment (ravindk89) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (ravindk89) [reply README.md]: Done

**Review by dvaldivia — APPROVED** [review]

**Review by nitisht — APPROVED** [review]

**Pull request merged**